docs/sections/user_guide/index.rst

```diff
@@ -7,20 +7,20 @@ User Guide
    installation
    cli/index
    api/index
+   yaml/index
 
 .. raw:: html
 
    <ul>
-   <li>API Jupyter Notebooks
-      <ul>
-         <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fconfig.ipynb">Config Tool</a>
-         <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Ffs.ipynb">File System Tool</a>
-         <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Frocoto.ipynb">Rocoto Tool</a>
-         <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Ftemplate.ipynb">Template Tool</a>
-      </ul>
+      <li>Jupyter Notebook Tutorials (API)
+         <ul>
+            <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fconfig.ipynb">Config Tool</a>
+            <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Ffs.ipynb">File System Tool</a>
+            <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Frocoto.ipynb">Rocoto Tool</a>
+            <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Ftemplate.ipynb">Template Tool</a>
+         </ul>
+      <li>Cookbooks
+         <ul>
+            <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fconfig-exp-cookbook.ipynb">Configuring an Experiment with UW Tools</a>
+         </ul>
```

Comment:
The [Issue](https://github.com/ufs-community/uwtools/issues/631) for this notebook mentions perhaps adding a link to the top-level index of the docs. I think it makes sense to include notebook links to the user-guide index, but let me know if I should either move these to the top-level index or create duplicate links there.

Reply:
I think it might be nice to move these links out to an additional `.rst` file, similar to how `installation.rst` is handled. Then you can reference it both here and in the top-level index intro page.

Reply:
That sounds good to me, I added a `binder_links.rst` file for the links and added them to both the top-level index and user-guide index.